stereovision/ui_utils.py

```diff
@@ -136,8 +136,9 @@ def _set_value(self, parameter, new_value):
         try:
             self.block_matcher.__setattr__(parameter, new_value)
         except BadBlockMatcherArgumentError:
+            print('Bad Block')
             return
-        self.update_disparity_map()
+        #self.update_disparity_map()
```

Comment:
Hi ! My changes are a bit messy: lots of things are specific to my needs and camera, but a few might be of general interest. Don't hesitate to ask, I could try shedding some light on them and how to make it cleaner.